moveit_core/kinematics_base/src/kinematics_base.cpp

```diff
@@ -112,15 +112,15 @@ bool KinematicsBase::initialize(const std::string& robot_description, const std:
     return initialize(robot_description, group_name, base_frame, tip_frames[0], search_discretization);
   }
 
-  ROS_ERROR_NAMED(LOGNAME, "This solver does not support multiple tip frames");
+  ROS_ERROR_NAMED(LOGNAME.c_str(), "This solver does not support multiple tip frames");
```

Comment:
Perhaps it would be better to save LOGNAME as a C string to begin with or to change all of the logging to use the new ROS2 logging functionality.

Reply:
Total guess here, but this logging functionality is likely all C code...

Reply:
If you use the `RCLCPP_ERROR` version it will take a `std::string`. The `RCUTILS_ERROR` macro is pure C, so it cannot take a C++ type.

See: https://github.com/ros-planning/moveit2/pull/7#discussion_r258702364

Reply:
43a77cf2ad36201607eeee8f780bc853ad42827a